README.md

```diff
@@ -16,13 +16,14 @@ The project is in the early development stage. More description will be added la
 
 ## Getting started
 
-Take a look on examples of how to run PipelineDP Apache Spark or Beam:
+Take a look on examples of how to run PipelineDP:
 
 * [Apache Spark example](examples/movie_view_ratings_spark.py)
 * [Apache Beam example](examples/movie_view_ratings_beam.py)
-* [Local execution example](examples/movie_view_ratings.py)
+* [W/o frameworks example](examples/movie_view_ratings_local.py)
```

Comment:
nit: "Framework-free example" or "Example without using any frameworks" (which is similar to the next point)